firmware/cockroach_main/cockroach_main.ino

```diff
@@ -0,0 +1,98 @@
+/*  Daniel Fearn for CUSF 30/03/21 */
+
+/*  Project Cockroach
+
+    SD card SPI bus connected as follows:
+    * SS - pin 10
+    * MOSI - pin 11
+    * MISO - pin 12
+    * SCK - pin 13
+
+    MAX-M8Q gps connected as follows:
+    * TP - pin 5 (unused?)
+    * RX0 - pin 1 / TX
+    * TX0 - pin 0 / RX (default Serial pins)
+
+    Summary:
+    * Upon startup a new file named data_<index>.csv is created, where <index> is the smallest integer not already in use
```

Comment:
Might be good to create all these files in a folder called `data`, for example, then make sure you number them with several leading 0s so alphabetical sorting displays them in the correct numerical order.